.github/workflows/workspace.yml

```diff
@@ -40,49 +40,49 @@ jobs:
           command: fmt
           args: --all -- --check
 
-  codecov:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Cache cargo registry
-        uses: actions/cache@v1
-        with:
-          path: ~/.cargo/registry
-          key: ${{ runner.os }}-cargo-registry-${{ hashFiles('Cargo.lock') }}
-
-      - name: Cache cargo index
-        uses: actions/cache@v1
-        with:
-          path: ~/.cargo/git
-          key: ${{ runner.os }}-cargo-index-${{ hashFiles('Cargo.lock') }}
-
-      - name: Cache cargo build
-        uses: actions/cache@v1
-        with:
-          path: target
-          key: ${{ runner.os }}-coverage-cargo-build-target-${{ hashFiles('Cargo.lock') }}
-
-      - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
-
-      - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@v0.1
-        env:
-          CARGO_INCREMENTAL: 0
-        with:
-          version: 0.14.2
-          args: --all -- --test-threads 1
-
-      - name: Upload to codecov.io
-        uses: codecov/codecov-action@v1.0.10
-
-      - name: Archive code coverage results
-        uses: actions/upload-artifact@v1
-        with:
-          name: code-coverage-report
-          path: cobertura.xml
+#  codecov:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - name: Checkout repository
+#        uses: actions/checkout@v2
+#
+#      - name: Cache cargo registry
+#        uses: actions/cache@v1
+#        with:
+#          path: ~/.cargo/registry
+#          key: ${{ runner.os }}-cargo-registry-${{ hashFiles('Cargo.lock') }}
+#
+#      - name: Cache cargo index
+#        uses: actions/cache@v1
+#        with:
+#          path: ~/.cargo/git
+#          key: ${{ runner.os }}-cargo-index-${{ hashFiles('Cargo.lock') }}
+#
+#      - name: Cache cargo build
+#        uses: actions/cache@v1
+#        with:
+#          path: target
+#          key: ${{ runner.os }}-coverage-cargo-build-target-${{ hashFiles('Cargo.lock') }}
+#
+#      - name: Install stable toolchain
+#        uses: actions-rs/toolchain@v1
+#        with:
+#          toolchain: stable
+#          override: true
+#
+#      - name: Run cargo-tarpaulin
+#        uses: actions-rs/tarpaulin@v0.1
+#        env:
+#          CARGO_INCREMENTAL: 0
+#        with:
+#          version: 0.14.2
+#          args: --all -- --test-threads 1
+#
+#      - name: Upload to codecov.io
+#        uses: codecov/codecov-action@v1.0.10
+#
+#      - name: Archive code coverage results
+#        uses: actions/upload-artifact@v1
+#        with:
+#          name: code-coverage-report
+#          path: cobertura.xml
```

Comment:
Not sure why but `cargo-tarpaulin` is failing:

https://github.com/RustCrypto/elliptic-curves/pull/85/checks?check_run_id=904067088

I attempted to reproduce it locally but cannot. It doesn't appear to be cache-related as this commit is a cache breaker due to the `Cargo.lock` changes.

I'd like to do a pass over `p256` to get it in to a similar state as this PR (as part of #84) after which I can try re-enabling this.